Pilot/PilotLogger.py

```diff
@@ -25,9 +26,9 @@ def connect(host_and_port, ssl_cfg):
   try:
     connection = stomp.Connection(host_and_ports=host_and_port, use_ssl=True)
     connection.set_ssl(for_hosts=host_and_port,
-                       key_file=ssl_cfg['key_file'],
-                       cert_file=ssl_cfg['cert_file'],
-                       ca_certs=ssl_cfg['ca_certs'])
+                       key_file=ssl_cfg.get('key_file'),
```

Comment:
Why ? isn't it supposed to be always there ?

Reply:
I am not sure about it. @wkrzemien will take care of this.